js/admin.js

```diff
@@ -0,0 +1,31 @@
+jQuery( document ).ready(function($) {
+	$('#check-status').click(function(){
+		$(".theme-check-report").empty();
+		themeCheckRunPHPCS();
+	});
+	function themeCheckRunPHPCS(){
+		$.ajax({
+			url: ajaxurl,
+			type: 'POST',
+			data: {
+				'action': 'ns_theme_check_run',
+				'themename': $('select[name=themename]').val(),
+				'hide_warning': $('input[name=hide_warning]').is(':checked'),
+				'raw_output': $('input[name=raw_output]').is(':checked'),
+				'minimum_php_version': $('select[name=minimum_php_version]').val(),
+				'wordpress-theme': $('input[name=wordpress-theme]').is(':checked'),
+				'wordpress-core': $('input[name=wordpress-core]').is(':checked'),
+				'wordpress-extra': $('input[name=wordpress-extra]').is(':checked'),
+				'wordpress-docs': $('input[name=wordpress-docs]').is(':checked'),
+				'wordpress-vip': $('input[name=wordpress-vip]').is(':checked'),
+				'ns_theme_check_nonce': $('#ns_theme_check_nonce').val(),
+			},
+			success:function(data) {
+				$(".theme-check-report").html(data);
+			},
+			error: function(errorThrown){
+				console.log(errorThrown);
+			}
+		});
+	}
+});
```

Comment:
I have VVV on my computer and run it there. The plugin is compiled by hand using bash for windows (on windows 10). Also when I did `error_log` I got something like this:

~~~
[01-Apr-2017 20:42:39 UTC] flexible-lite
[01-Apr-2017 20:42:39 UTC] Array
(
    [minimum_php_version] => 5.4
    [standard] => Array
        (
            [0] => WordPress-Theme
            [1] => WordPress-Core
            [2] => WordPress-Extra
            [3] => WordPress-Docs
            [4] => WordPress-VIP
        )

    [text_domains] => Array
        (
            [0] => flexible-lite
        )

)
~~~

So the callback definitely works. On a theme where I deleted tags from `style.css` I got the errors about those not being set, but any of the styling errors, or escape errors or such weren't shown.

And there were such errors, because when I did `phpcs` command from my CLI I got them all. So I don't know what could be the issue :\